pipx/pipxrc.py

```diff
@@ -4,6 +4,16 @@
 from pipx.Venv import PipxVenvMetadata
 
 
+# handy, helps enforce some consistency, and adds pipxrc_version
+pipxrc_info_template = {
+    "package_or_url": None,
+    "install": {"pip_args": [], "venv_args": [], "include_dependencies": None},
+    "venv_metadata": None,
+    "injected_packages": {},
+    "pipxrc_version": 0.1,
```

Comment:
It's more common to store versions as strings (.e.g https://www.python.org/dev/peps/pep-0396/#specification); it's a bit more consistent. `List[int]` would also work although maybe it's better to just parse that out of a string.

Reply:
Sounds good.  I'm guessing this is especially for allowing minor version numbers (not proper floats).

Reply:
Exactly - it also gives you the option of doing major.minor.patch-devN etc., not that we need to go that far here.